src/main/java/Milo/TaskObj/TaskType.java

```diff
@@ -0,0 +1,15 @@
+package Milo.TaskObj;
+
+/*
+ * Represents the different possible task type of the task object
+ */
+public class TaskType {
```

Comment:
Is there any reason why you wrap the `enum` inside the `TaskType` class instead of having just `public enum TaskType`?